### ISSUE TITLE
Rule Proposal: repetitive CSS selectors

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#44]
This is the follow-up to #25 but specific to CSS selectors (decided to make a separate rule):

Example - the locator of a parent element is repeated for multiple page object fields:

``` js
var MyPage = function () {
   this.parent = $(".container #parent");
   this.child1 = $(".container #parent div:first-of-type");
   this.child2 = $(".container #parent #subcontainer > .add-client");
}
```

The `.container #parent` part in this case is repeated and should be reused:

``` js
var MyPage = function () {
   this.parent = $(".container #parent");
   this.child1 = this.parent.$("div:first-of-type");
   this.child2 = this.parent.$("#subcontainer > .add-client");
}
```

NOTE: no violation should be reported if a _direct parent-child relationship_ detected after a repetitive part. E.g. `.container #parent > div:first-of-type` instead of a `.container #parent div:first-of-type` in the example above

**Comment by alecxe** [comment]
Closed by 3173ef37979e5670acca5d9f2035313c62a5f6ca. Though I feel this rule would need to be improved in the future to avoid false positives.
